src/WormholeJoin.sol

```diff
@@ -191,6 +192,8 @@ contract WormholeJoin {
             // amtToGenerate doesn't need overflow check as it is bounded by amtToTake
             vat.slip(ilk, address(this), int256(amtToGenerate));
             vat.frob(ilk, address(this), address(this), address(this), int256(amtToGenerate), int256(amtToGenerate));
+            (, uint256 art_) = vat.urns(ilk, address(this)); // Query the actual value as someone might have repaid debt without going through settle
```

Comment:
is this path impossible to reach when vatLive === 0?

Reply:
yes, vat.frob is blocked on shutdown

Reply:
True. I think this deserves a comment @gbalabasquer

Reply:
Clarification added